assets/css/headerAlt.css

```diff
@@ -15,12 +15,6 @@
     margin-right: 7px;
 }
 
-.container {
-    padding-left: 15px;
-    padding-right: 30px;
-    width: auto;
-}
```

Comment:
This styling was inadvertently affecting containers outside of the Header Alt and "masking" our issue.  I don't *think* there's any major ramifications for removing it, but we should be on the lookout.